Sources/Redis+Basic.swift

```diff
@@ -58,6 +58,88 @@ extension Redis {
+    /// (necessary) aligned offset.
+    /// https://redis.io/commands/bitfield
+    ///
+    /// - parameter key: The key of the string to manipulate.
+    /// - parameter subcommands: `BitfieldSubcommand`s to do on the string.
+    /// - parameter callback: The callback function.
+    /// - parameter res: An array with each entry being the corresponding result 
+    ///                  of the sub command given at the same position. OVERFLOW 
+    ///                  subcommands don't count as generating a reply.
+    /// - parameter err: The error, if one occurred.
+    public func bitfield(key: String, subcommands: BitfieldSubcommand..., callback: (_ res: [RedisResponse?]?, _ err: NSError?) -> Void) {
+        bitfieldArrayOfSubcommands(key: key, subcommands: subcommands, callback: callback)
+    }
+    
+    /// Treats a Redis string as a array of bits, and is capable of addressing
+    /// specific integer fields of varying bit widths and arbitrary non
+    /// (necessary) aligned offset.
+    /// https://redis.io/commands/bitfield
+    ///
+    /// - parameter key: The key of the string to manipulate.
+    /// - parameter subcommands: `BitfieldSubcommand`s to do on the string.
+    /// - parameter callback: The callback function.
+    /// - parameter res: An array with each entry being the corresponding result
+    ///                  of the sub command given at the same position. OVERFLOW
+    ///                  subcommands don't count as generating a reply.
+    /// - parameter err: The error, if one occurred.
+    public func bitfieldArrayOfSubcommands(key: String, subcommands: [BitfieldSubcommand], callback: (_ res: [RedisResponse?]?, _ err: NSError?) -> Void) {
+        var command = ["BITFIELD", key]
+        for subcommand in subcommands {
+            switch(subcommand) {
+            case .get(let type, let offset):
+                command.append("GET")
+                command.append(String(type))
+                command.append(String(offset))
+            case .set(let type, let offset, let value):
+                command.append("SET")
+                command.append(type)
+                command.append(String(offset))
+                command.append(String(value))
+            case .incrby(let type, let offset, let incr):
+                command.append("INCRBY")
+                command.append(type)
+                command.append(String(offset))
+                command.append(String(incr))
+            case .overflow(let overflow):
+                command.append("OVERFLOW")
+                switch(overflow) {
+                case .WRAP:
+                    command.append(BitfieldSubcommand.BitfieldOverflow.WRAP.rawValue)
+                case .SAT:
+                    command.append(BitfieldSubcommand.BitfieldOverflow.SAT.rawValue)
+                case .FAIL:
+                    command.append(BitfieldSubcommand.BitfieldOverflow.FAIL.rawValue)
+                }
```

Comment:
Why do you need the switch here at all? Why not just write:

     command.append(overflow.rawValue)

Isn't overflow of the type `BitfieldSubcommand.BitfieldOverflow`?